src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.scss

```diff
@@ -3,3 +3,22 @@
 .AnnotationThreadContent-loading {
     padding: $bdl-grid-unit * 5 0;
 }
+
+.AnnotationThread {
```

Comment:
Should this be in `AnnotationThread.scss`?

Reply:
Makes sense. I've moved it.